apps/backend/user/src/logout.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, Response } from "express";
+import { IDatabase } from "pg-promise";
+
+const logout = async (req: Request, res: Response, db: IDatabase<object>) => {
```

Comment:
Is this being left for resops to finish?

Reply:
correct